lib/src/main/scala/spinal/lib/bus/amba4/axi/Axi4.scala

```diff
@@ -298,6 +326,91 @@ object Axi4{
     def DECERR = B"11" // Decode error
   }
 
+  // ARM IHI 0022E, Table C3-7
+  object arsnoop {
```

Comment:
Arsnoop ?

Reply:
Apart from the AC/CR/CD snoop channels for a **snooped** master, ACE and ACE-Lite also adds ARSNOOP and AWSNOOP signals for a **snooping** master.  This (along with the `awsnoop` object) define encodings for these fields.